lib/selector/parser/ast.go

```diff
@@ -86,22 +86,22 @@ func (node LabelEqValueNode) collectFragments(fragments []string) []string {
 
 type LabelInSetNode struct {
 	LabelName string
-	Value     map[string]bool
+	Value     StringSet
 }
 
 func (node LabelInSetNode) Evaluate(labels map[string]string) bool {
 	if val, ok := labels[node.LabelName]; ok {
-		return node.Value[val]
+		return node.Value.Contains(val)
 	} else {
 		return false
 	}
 }
 
-func (node LabelInSetNode) collectFragments(fragments []string) []string {
+func collectInFragments(fragments []string, labelName, op string, values StringSet) []string {
```

Comment:
What does the "In" refer to in the function name - is it the operator which is actually passed in as an argument?  just wondering if collectInFragments and collectFragments are the wrong way round.